backend/tournesol/views/preview.py

```diff
@@ -0,0 +1,50 @@
+from PIL import Image, ImageDraw, ImageFont
+import requests
+from io import BytesIO
+from django.http import HttpResponse, FileResponse
+from django.utils.decorators import method_decorator
+from drf_spectacular.utils import OpenApiTypes, extend_schema
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.permissions import AllowAny, IsAdminUser, IsAuthenticated
+from rest_framework.views import APIView
+
+POPPINS_FONT = ImageFont.truetype("../frontend/public/fonts/Poppins-Bold.ttf", 40)
```

Comment:
This relative path will not work on the dev-env (as the CI shows) or on deployed environments, as "frontend" and "backend" are assumed to run independently. On the dev-env, "frontend" and "backend" live in separate volumes, and on the VMs they are deployed in distinct folders: "/srv/tournesol-frontend" and "/srv/tournesol-backend".

If this font file becomes a dependency of the API, I would recommend to copy it somewhere into "backend".

Reply:
Copied the necessary files to `backend/tournesol/resources/`

Whe ndeploying I found it worked for starting the server but failed during the stage "Migrate Django database" becasue the font was evaluated when importing the file. I guess this is because this step is done from another folder? Circumvented this for now by avoidnig to evaluate the font when the file is imported.

Reply:
To make it more robust you can use `BASE_DIR` (from django settings) which points to the "backend" content.
E.g:
```py
fnt = ImageFont.truetype(BASE_DIR / "tournesol/resources/Poppins-Medium.ttf", 20)
```